:warning: stabilize go/v4 plugins. Plugin go/v4-alpha replaced with go/v4 (#3237)
kubernetes-sigs/kubebuilder: README.md

### Apple Silicon

> Apple Silicon (`darwin/arm64`) is supported from the `go/v4` plugin which provides support for this platform. Note that
> previous plugin versions to scaffold the projects does not provide this support.

Reviewer: Nit:
```suggestion
Apple Silicon (`darwin/arm64`) support begins with the `go/v4` plugin.
```
Just feel like this is a bit more succinct.
